uber/templates/base.html

```diff
@@ -16,6 +16,7 @@
         var csrf_token = '{{ c.CSRF_TOKEN }}';
     </script>
     <script src="../static/js/common-static.js" type="text/javascript"></script>
+    <script src="../static/js/jquery.form.js" type="text/javascript"></script>
```

Comment:
In general, we should compile this into ``common-static.js``, which is our consolidated js file which has all of the Javascript dependencies in one place so that we can minimize the number of HTML requests we make for performance reasons.

Helpfully, I set that up and then didn't finish documenting it.  The PR where it was set up (where this is somewhat haphazardly explained) is at
https://github.com/magfest/ubersystem/pull/1105
and the open issue for finishing the docs (which contain some of the docs) is at #1104